src/org/labkey/test/util/SampleTypeHelper.java

```diff
@@ -78,6 +78,10 @@ public void createSampleType(SampleTypeDefinition props)
         {
             createPage.setAutoLinkDataToStudy(props.getAutoLinkDataToStudy());
         }
+        if(props.getLinkedDatasetCategory() != null)
```

Comment:
I think for `props.getAutoLinkDataToStudy()` and `props.getLinkedDatasetCategory()` the if should be updated with a space so it looks like `if (<condition>)` instead of `if(<condition>)`